src/data-display/tags/PTag.vue

```diff
@@ -112,12 +136,12 @@ export default defineComponent<Props>({
             @apply bg-red-200 border-red-500;
         }
     }
-    .text {
-        font-size: 0.75rem;
-        line-height: 1.4;
-        white-space: normal;
-        word-break: break-word;
-        max-width: 100%;
```

Comment:
I think removing these css attributes can make some side effects.
Such as text overflow. Is it OK ?

Reply:
Thanks, i'll handle that.